src/NetscriptFunctions/Gang.ts

```diff
@@ -134,17 +167,28 @@ export function NetscriptGang(): InternalAPI<IGang> {
       const gang = getGang(ctx);
       return gang.canRecruitMember();
     },
+    getRecruitsAvailable: (ctx) => () => {
+      const gang = getGang(ctx);
+      return gang.getRecruitsAvailable();
+    },
     recruitMember: (ctx) => (_memberName) => {
       const memberName = helpers.string(ctx, "memberName", _memberName);
       const gang = getGang(ctx);
       const recruited = gang.recruitMember(memberName);
-      if (recruited) {
+      // TODO : ns.gang.recruitMember() crashes with error, can use top return instead?
```

Comment:
This TODO comment isn't relevant really, it's the same behavior as any bad call missing args. Thought I had removed this line.